### ISSUE TITLE
Negative tab indices should not be tabbable

### ISSUE THREAD

**Issue by tinynow** [davidtheclark/tabbable#18]
> A negative value (usually tabindex="-1") means that the element should be focusable, but should not be reachable via sequential keyboard navigation. Mostly useful to create accessible widgets with JavaScript.
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

This could be solved by changing `'[tabindex]',`  to `'[tabindex]:not([tabindex*="-"])',`

**Comment by davidtheclark** [comment]
@tinynow I believe this is already addressed by https://github.com/davidtheclark/tabbable/blob/166909a25f4f449108b25d47606f9f578dc78164/index.js#L42, where we ignore nodes whose tabindex is less than 0.

**Comment by tinynow** [comment]
Sorry, I just noticed that.  Thank you.
